explorer-ui/src/components/MatchGrid/MatchGrid.jsx

```diff
@@ -16,119 +16,158 @@ import { Map, GoogleApiWrapper, InfoWindow, Marker } from "google-maps-react";
+//       if (matches) {
+//         for (let i = 0; i < matches.length; i++) {
+//           if(matches[i].address != null || matches[i].address !="" ){
           
-          Geocode.fromAddress(matches[i].address).then(
-            (response) => {
-              const { lat, lng } = response.results[0].geometry.location;
-              setMatchesPlusCoordinates([...matchesPlusCoordinates, {id: i, position: {lat, lng}, user: matches[i]}])
+//           Geocode.fromAddress(matches[i].address).then(
+//             (response) => {
+//               const { lat, lng } = response.results[0].geometry.location;
+//               setMatchesPlusCoordinates([...matchesPlusCoordinates, {id: i, position: {lat, lng}, user: matches[i]}])
               
-            },
-            (error) => {
-              console.error(error);
-            }
-          );
-        }
-        }
-        // Get latitude & longitude from address.
-      }
-    };
-    const getUserCoords = (user) => 
-    {Geocode.fromAddress(user.address).then(
-      (response) => {
-        const { lat, lng } = response.results[0].geometry.location;
-        setUserCoordinates( {id: 0, position: {lat, lng}, user: user})
-      },
-      (error) => {
-        console.error(error);
-      }
-    );
-  }
+//             },
+//             (error) => {
+//               console.error(error);
+//             }
+//           );
+//         }
+//         }
+//         // Get latitude & longitude from address.
+//       }
+//     };
+//     const getUserCoords = (user) => 
+//     {Geocode.fromAddress(user.address).then(
+//       (response) => {
+//         const { lat, lng } = response.results[0].geometry.location;
+//         setUserCoordinates( {id: 0, position: {lat, lng}, user: user})
+//       },
+//       (error) => {
+//         console.error(error);
+//       }
+//     );
+//   }
 //   const { isLoaded } = useLoadScript({
 //     googleMapsApiKey: "", // Add your API key
 //   });
 
 
+  // return (
+  //   <div className="match-grid">
+  //     <h1>Here are your matches!</h1>
+  //     {matches != [] ? (
+  //       matches.map((sortedMatch, idx) => {
+  //         return (
+  //           <MatchCard
+  //             name={sortedMatch.match.name}
+  //             bio={sortedMatch.match.bio}
+  //             picture={sortedMatch.match.picture}
+  //             country={sortedMatch.match.country}
+  //             accomodation={sortedMatch.match.accomodation}
+  //             key={idx}
+  //           />
+  //         );
+  //       })
+  //     ) : (
+  //         <h1> Sorry we couldn't find any matches for you!</h1>
+  //     )}
+  //   </div>
+  // );
+
+
   return (
     <div className="match-grid">
+      {loadingComplete
+      ? 
+      <div>
       <h1>Here are your matches!</h1>
-      {sorted != [] ? (
-        sorted.map((sortedMatch, idx) => {
+      {matches != [] ? (
+        matches.map((sortedMatch, idx) => {
           return (
```

Comment:
nit - can remove the return and do () => <MatchCard />

Reply:
seems to be causing syntax errors when i try